src/compiler/commandLineParser.ts

```diff
@@ -2913,6 +2917,11 @@ namespace ts {
             }
         }
 
+        // eslint-disable-next-line no-in-operator
+        if (rootCompilerOptions && json && !("compilerOptions" in json)) {

```

Comment:
You could write `json.compilerOptions === undefined` and avoid the linter suppression, right?

Reply:
There’s also a `hasProperty` util